shpkpr/cli/options.py

```diff
@@ -110,11 +113,21 @@
 )
 
 
-template_file = click.option(
+template_path = click.option(
+    '--template_dir',
+    'template_path',
+    envvar="{0}_TEMPLATE_DIR".format(CONTEXT_SETTINGS['auto_envvar_prefix']),
```

Comment:
NAB: Every time I see {0} I have to think.. Just a personal reference. You could leave it for {}_TEMPLATE_DIR - but this is just a personal preference.